### PR TITLE
Basic Metrics Test

### PR DESCRIPTION
**Pull request by pavelzag**
Purpose or Intent
=================

This is a basic test that checks if Metrics for OSE are up
{{pytest: cfme/tests/containers/test_basic_metrics.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a6e2b5c4ac910ee5a1eedf974f75c48496068593:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_basic_metrics.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+def test_basic_metrics(provider):
+    """ Basic Metrics availability test
+        This test checks that the Metrics service is up
+        Opens the hawkular status page and checks if it's up
+        """
+    hostname = 'https://' + conf.cfme_data.get('management_systems', {})[provider.key]\
+        .get('hostname', []) + '/hawkular/metrics'
+    ensure_browser_open()
```
Comment (psav) [inline cfme/tests/containers/test_basic_metrics.py]: This shouldn't be called inside a test ever

Comment (psav) [inline cfme/tests/containers/test_basic_metrics.py]: It probably makes sense to make a method on the provider object called ```check_metrics``` or something like that

Comment (psav) [inline cfme/tests/containers/test_basic_metrics.py]: Or you could use the browser fixture inside another fixture to obtain the right browser location

Comment (pavelzag) [inline cfme/tests/containers/test_basic_metrics.py]: I could add it to a provider object but I fear it would take more time to go through approval :D

Comment (mfalesni) [inline cfme/tests/containers/test_basic_metrics.py]: That is the point of the approval, that the code is right ...

Comment (pavelzag) [inline cfme/tests/containers/test_basic_metrics.py]: Yes.. I guess

**Comment by jkrocil** [comment]
Sorry for jumping in but:
```
Basic Metrics availability test
  This test checks that the Metrics service is up
  Opens the hawkular status page and checks if it's up
```
What does this have to do with ManageIQ testing at all? Am I missing something here?
If you are writing a test against ManageIQ and you want to make sure that your infrastructure is running, you can check that as part of a fixture for the real test. But I don't see the "real test" anywhere.
What I see here is just a fixture checking environment's availability wrapped as a test and that's it.

**Comment by pavelzag** [comment]
The Metrics part of the OSE is a part that breaks often. We'd like to have heads up every time sometimes in deployment changes and the Metrics system is down. We wasted a lot of time in the past to find what went wrong and I proposed that we'd have a simple test that checks if Hawkular is up and Metrics is working.
It's a super simple test but believe me it'll spare us majore headaches going forward

**Comment by pavelzag** [comment]
Added a check_metrics function to provider object. Please let me know if it's better

**Comment by dajoRH** [comment]
Lint report for commit 634f5a1840dabc98a57e95eb83a329ac52b28bd5:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by jkrocil** [comment]
@pavelzag #3410 alredy got merged so what is the point of this anyway? Like I said before, this can be a fixture checking environment availability right before the test itself but thats about it.

**Comment by pavelzag** [comment]
@jkrocil This change adds check_metrics function in provider's __init__.py per request. 
I'd like this to remain a test so I can see in Jenkins' reports if Metrics is up or down

**Comment by jkrocil** [comment]
But this functionality is already present in master, see here: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/containers/test_basic_metrics.py
I prefer to have this code as a test the way you've already merged it rather than having it as part of provider.

**Comment by pavelzag** [comment]
@jkrocil I have no problems with that. Thanks

**Comment by mfalesni** [comment]
*The Metrics part of the OSE is a part that breaks often.* - I would not say that in public :D

**Comment by pavelzag** [comment]
True, let's give it a codename to confuse the enemy. :D

**Pull request closed without merging**